platforms/intellij/compose/playground/build.gradle.kts

```diff
@@ -56,3 +76,12 @@ configurations
   .configureEach { attributes { attribute(KotlinPlatformType.attribute, KotlinPlatformType.jvm) } }
 
 dependencies { lintChecks(libs.composeLints) }
+
+// Roborazzi Desktop support uses Context Receivers
+// This setup is for JVM test builds only
+tasks.named<KotlinCompile>("compileTestKotlinJvm").configure {
+  compilerOptions {
+    incremental = false
```

Comment:
What's this for?

Reply:
https://github.com/slackhq/foundry/pull/1064#discussion_r1819340674

I was trying to specify just needing the task name for "compileTestKotlinJvm"